spec/rails_helper.rb

```diff
@@ -31,14 +31,22 @@ module Features
 end
 
 ActiveRecord::Migration.maintain_test_schema!
-Capybara.javascript_driver = :poltergeist
-Capybara.register_driver :poltergeist do |app|
-  options = { phantomjs_options: ["--load-images=no"] }
-  Capybara::Poltergeist::Driver.new(app, options)
+
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(app, browser: :chrome)
 end
 
-Capybara.register_driver :rack_test do |app|
-  Capybara::RackTest::Driver.new(app, respect_data_method: false)
+Capybara.register_driver :headless_chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: {
+      args: %w[headless enable-features=NetworkService,NetworkServiceInProcess]
+    }
+  )
+  Capybara::Selenium::Driver.new app,
+                                 browser: :chrome,
+                                 desired_capabilities: capabilities
```

Comment:
I find this style of indent problematic. It breaks the indent for all lines if the first one is replaced with one of different length. How about writing it like this?

```suggestion
  Capybara::Selenium::Driver.new(
    app,
    browser: :chrome,
    desired_capabilities: capabilities,
)
```

To be honest, I'm not sure if we have a style rule for this, and Hound appears not to be enabled at the moment? @nickcharlton, do you have any thoughts?

Reply:
I'm curious about the `tmp/pids/.keep`. I tried to fix the first clone problem you'd get in #1695, do we need to ignore that file too so if you delete it locally it doesn't matter?

Hound did run with this PR. It might've been done when the configuration in `thoughtbot/guides` was deleted. It got put back in again, so we might be alright now. A new push might highlight the changes.

Reply:
@pablobm This indentation style was suggested by Houndbot, as was many of the changes you and Nick asked me to undo in this commit. I like yours better though!

Reply:
Oh, sorry about that. Normally we follow Hound's opinion to the letter, but a couple of weeks ago there was an issue with it. The set of rules changed and I'm not sure they are back to normal yet.

Don't worry then. Go with your preference and we'll come around to fixing Hound in the future.

Reply:
Yeah, it’s been really annoying. I don’t much like commenting on style things manually, as even though consistency is good, nit-picking to the nth degree is not enjoyable for anyone!

I think there’s only one or two things left here: everything we can fix on GitHub apart from the `tmp/pids/.keep` file missing.

Reply:
Okay, I think I've fixed all the issues now from the houndbot to the tmp/pids/.keep (which was in the gitignore so wasn't getting pushed). Thanks @nickcharlton